index.html

```diff
@@ -61,9 +60,152 @@ <h1>I’m Alexandra.   Glad   to  see      you!</h1>
+                            <a class="button">See project</a>
+                        </div>
+                    </section>
+            
+
+                    <section class="myproject">
+                        <div class="pheader">
+                            <img src="./img/projectthree.png">
+                            <h3 class="ptitle">Tonic</h3>
+                        </div>
+                        <div class="p-details">
+                            <ul>
+                                <li>CANOPY</li>
+                                
+                                <li>Back End Dev</li>
+                               
+                                <li>2015</li>
+                            </ul>
+                        </div>
+                        <p class="">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <div class="lower_part">
+                            <ul class="project-tech-stack">
+                                <li><a>HTML</a></li>
+                                <li><a>CSS</a></li>
+                                <li><a>Javascript</a></li>
+                            </ul>
+                            <a class="button">See project</a>
+                        </div>
+                    </section>
+                    <section class="myproject">
+                        <div class="pheader">
+                            <img src="./img/projectfour.png">
+                            <h3 class="ptitle">Multi-Post Stories</h3>
+                        </div>
+                        <div class="p-details">
+                            <ul>
+                                <li>CANOPY</li>
+                                
+                                <li>Back End Dev</li>
+                               
+                                <li>2015</li>
+                            </ul>
+                        </div>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <div class="lower_part">
+                        <ul class="project-tech-stack">
+                            <li><a>HTML</a></li>
+                            <li><a>CSS</a></li>
+                            <li><a>Javascript</a></li>
+                        </ul>
+                        <a class="button">See project</a>
+                        </div>
+                    </section>
+            </section>
+            
+            
+            <section class="about-me">
+                <h2 class="titletext">About Myself</h2>
+                <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and 
+                    experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+                </p>
+                <p class="letconnect"> LET’S CONNECT</p>
```

Comment:
Great effort in adding the section subtitle! However, there are a few adjustments needed to align it with the design:

- [ ] Please ensure that the font styles, such as font family, font size, and font weight, match the design specifications. Consistency in font styles will contribute to a cohesive and visually appealing design.

- [ ] Pay attention to the spacing and alignment of the section subtitle. Make sure it is properly aligned with the surrounding elements and that the spacing matches the design guidelines. Proper spacing will help create a well-structured and balanced layout.

- [ ] Consider using an appropriate heading tag, such as `<h2>`, for the section subtitle instead of a `<p>` tag. This will provide semantic meaning to the element and improve accessibility.

- [ ] Check the colors used for the section subtitle and ensure they are consistent with the design. Use the appropriate color codes or classes to achieve the desired look.

By updating the font styles, spacing, colors, and using an appropriate heading tag for the section subtitle, you will bring it in line with the design and enhance the overall visual appeal of the webpage. Keep up the good work and strive for design accuracy!